README.md

```diff
@@ -39,18 +39,18 @@ or starting a container manually:
 $ docker run --detach \
              --name thelounge \
              --publish 9000:9000 \
-             --volume ~/.thelounge:/var/opt/thelounge \
+             --volume thelounge:/var/opt/thelounge \
```

Comment:
I think keeping the old `~/.thelounge` path could be helpful, rather than "creating" a host directory on whatever cwd the user runs this command on.

Reply:
That's not a local directory, that's a named volume (a local directory would have to be specified as `./thelounge`).

Reply:
Ah of course!